andela/travel_tool_front
# #162127512 Manager can approve or reject travel request (End-to_end-Test)

#### What does this PR do?
Adds end-to-end tests that ensure that manager can approve or reject travel request

#### Description of Task to be completed?
- Add test for approval modal
- Add test for approval and rejection of travel request

#### How should this be manually tested?
- Open the terminal  
- Create `cypress.env.json` file in the root directory and add your token following the format:
``{
"token": "your_token"
}``
- Run `yarn end2end` to run the test with browser mode
- When redirected to the browser, select `component/Approvals/approve-reject-request.spec.js`
- The tests should run and pass
- To run test only in the console, run `yarn end2end:headless` 

#### Any background context you want to provide?
- Ensure both the frontend and backend servers are running

#### What are the relevant pivotal tracker stories?
[#162127512](https://www.pivotaltracker.com/story/show/162127512)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
